src/abstractdataframe/io.jl

```diff
@@ -184,8 +184,13 @@ end
         write(io, "<tr>")
         write(io, "<th>$row</th>")
         for column_name in cnames
-            cell = string(df[row, column_name])
+          if isnull(df[row, column_name])
```

Comment:
I tried `ourshowcompact` but I think I might be missing something as I'm still seeing the quotes:
![image](https://cloud.githubusercontent.com/assets/7132414/19019070/6b4b6550-882d-11e6-8477-a56741b10722.png)